hardware-and-software-requirements.md

```diff
@@ -84,7 +84,7 @@ You can deploy and run TiDB on the 64-bit generic hardware server platform in th
 > - For the TiKV server, it is recommended to use NVMe SSDs to ensure faster reads and writes.
 > - If you only want to test and verify the features, follow [Quick Start Guide for TiDB](/quick-start-with-tidb.md) to deploy TiDB on a single machine.
 > - The TiDB server uses the disk to store server logs, so there are no special requirements for the disk type and capacity in the test environment.
-> - Starting from v6.3.0, to deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. By using such CPU instruction sets, TiFlash's vectorization engine can deliver better performance.
+> - Starting from v6.3.0, to deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. To deploy TiFlash under the Linux ARM64 architecture, the CPU must support ARMv8 instruction sets. Use `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` to confirm that there is output. By using such CPU instruction sets, TiFlash's vectorization engine can deliver better performance.
```

Comment:
```suggestion
> - Starting from v6.3.0, to deploy TiFlash under the Linux AMD64 architecture, the CPU must support AVX2 instruction sets. Use `cat /proc/cpuinfo | grep avx2` to confirm that there is output. To deploy TiFlash under the Linux ARM64 architecture, the CPU must support ARMv8 Instruction Set Architecture. Use `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` to confirm that there is output. By using such CPU instruction set extensions, TiFlash's vectorization engine can deliver better performance.
```

And "Use `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` to confirm that there is output." is kind of confusing. Maybe "Please make sure the command `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'" has output in your machine.` is better or another one.

Reply:
Fixed in d56b309. `AVX2 instruction sets` and `ARMv8 instruction sets` are kept unchanged because it seems `instruction sets` are commonly used and can be clearly understood without any misunderstandings.

Reply:
`ARMv8` is ISA(Instruction Set Architecture), while `AVX2` is Instruction Set.

Reply:
Fixed in d4869d4

Reply:
Only when we enable CPU instruction set extensions like `asimd` and `crc32` on `ARMv8`, tiflash can deliver better performance.  So maybe 

```
By using the instruction set extensions, TiFlash's vectorization engine can deliver better performance.
```
is better.

Reply:
@Lloyd-Pottiger Fixed in [3ea3458](https://github.com/pingcap/docs/pull/12386/commits/3ea3458af9aac541de4fd40c218a833a1b5cbe8b). BTW, do we need to synchronize the preceding changes to the [Chinese version](https://github.com/pingcap/docs-cn/pull/12812/files) as well? If yes, please help update the Chinese, thanks.

Reply:
+1

Reply:
done